docs/docs/sign-in-out.md

```diff
@@ -23,17 +36,29 @@ server.auth.sign_out()
     limited by the maximum session length (of four hours) on Tableau Server.
 </div>
 
-
-Alternatively, for short programs, consider using a `with` block:
+### Disabling certificate verification and warnings
+If a self-signed certificate is used, certificate verification can be disabled with ```add_http_options```:
```

Comment:
Self-signed isn't what's bad, necessarily, it's that they aren't trusted by the client.

The best way to phrase this might be something like:
```text
Certain environments may throw errors related to ssl configuration, such as self-signed certificates, expired certificates, etc...```